css/welcome.css

```diff
@@ -0,0 +1,258 @@
+    background: linear-gradient(45deg, #a7daf5, #ffffff, #a7daf5) padding-box,
+                linear-gradient(45deg, gold, #ffffff, gold) border-box;
+}
+
+#welcome-nav a:hover, #welcome-nav a:focus-visible {
+    box-shadow: 2px 2px 5px var(--accent-color), -2px -2px 5px var(--accent-color);
+}
+
+#loading-overlay {
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100vw;
+    height: 100vh;
+    background-color: var(--accent-color);
+
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+
+    transition: 0.5s transform ease-in, 0.5s opacity ease;
+    transform: translateY(-100%);
+    opacity: 0;
+    pointer-events: none;
+}
+
+#loading-overlay.active {
+    transform: translateY(0%);
+    opacity: 1;
+    pointer-events: auto;
+}
+
+#loading-overlay span {
+
+    margin: 0 0.15em;
+
+    font-style: italic;
+    font-size: 2em;
+    color: white;
+    text-shadow: 2px 2px 1px rgb(105, 158, 255);
+
+    animation-name: loading-overlay-loading-hop;
+    animation-timing-function: ease-in-out;
+    animation-duration: 0.8s;
+    animation-iteration-count: infinite;
+    animation-direction: alternate;
+    animation-fill-mode: forwards;
+
+    transform-origin: 50% 100%;
+}
+
+#loading-overlay span:nth-child(2) {
+    animation-delay: 0.25s;
+}
+#loading-overlay span:nth-child(3) {
+    animation-delay: 0.5s;
+}
+#loading-overlay span:nth-child(4) {
+    animation-delay: 0.75s;
+}
+
+@keyframes loading-overlay-loading-hop {
+    0% {
+        transform: translateY(0%);
+    }
+    100% {
+        transform: translateY(-50%);
+    }
+}
+
+@keyframes alt-float {
+    0% {
+        transform: translateY(-10px);
+    }
+    100% {
+        transform: translateY(10px);
+    }
+}
+
+@media only screen and (max-width: 600px) {
+    main {
+        position: absolute;
+        top: 0;
+        left: 0;
+        /* prevent horiz scrolling */
+        overflow-x: hidden; 
+    }
+
+    #altare-img {
+        top: calc(50% - 80vmin);
+        right: 0;
+        width: 100%;
+        max-height: 100vh;
+    }
+
+    #welcome-fancy-menu {
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100vh;
+    }
+
+    #welcome-nav {
+        position: absolute;
+        bottom: 1em;
+        right: 0;
+        top: auto;
+        height: 100%;
+        max-height: none;
+        padding: 2em;
+
+        box-sizing: border-box;
+        display: flex;
+        flex-direction: column;
+        gap: 0.5em;
+        align-items: flex-end;
+        justify-content: flex-end;
+    }
+
+    #welcome-nav a {
+        position: relative;
+        width: 50%;
+        height: auto;
+        transform: none;
+        padding: 0.75em 0.5em;
+    }
+}
+
+@media only screen and (min-width: 600px) {
+
+    #welcome-nav a:hover, #welcome-nav a:focus-visible {
+        transform: scale(1.1) rotate3d(0, 0, 0, 0deg) !important;
+    }
+    
+    #welcome-nav a:nth-child(1) {
```

Comment:
No changes needed now - FYI the best practice is to have buttons/ links ordered top/ down, left/ right to make it a predictable order for users tabbing through the page. In our context I think it's fine/ cute to break that rule for just the "Play Gacha" card, but I would expect the other cards' tab order to go left > right. Right now it appears to be going inside out

We can address this later when the design is more final